theseus/geometry/vector.py

```diff
@@ -42,6 +43,28 @@ def _init_data(dof: int) -> torch.Tensor:  # type: ignore
     def dof(self) -> int:
         return self.data.shape[1]
 
+    @staticmethod
+    def rand(
+        *size: int,
```

Comment:
In reference to the batch size comment above, it could get confusing here if we need both the batch and vector size.

Reply:
`*size` here is more than a `batch size`. For `theseus.Vector`, `*ize` could be both the `batch size` and `vector dimensions`, whereas for `theseus.SO3`, `size` only refers to the `batch size`.

Reply:
Okay thanks for clarifying. I think once we add documentation this stuff will be more obvious.